docs/js.md

```diff
@@ -216,8 +237,7 @@ println(node.innerHTML)
 ```scala mdoc:js
 <p>I am a custom <code>loader</code></p>
 ---
-// Open developer console to see this printed message
-println(s"Loading HTML: ${node.innerHTML}")
```

Comment:
Why the change here?

Reply:
I don't think it should have been changed, reverting :-(. Apologies.